# fix /usr/loca/bin/perl
# May 03rd 2018 by B. Hajgato (Free University Brussels - VUB)
diff -ru mcl-14-137.orig/doc/minimcl mcl-14-137/doc/minimcl
--- mcl-14-137.orig/doc/minimcl	2006-02-27 00:18:05.000000000 +0100
+++ mcl-14-137/doc/minimcl	2018-05-03 10:43:10.577298454 +0200
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/env perl -w
 
 #  (C) Copyright 2006 Stijn van Dongen
  #
diff -ru mcl-14-137.orig/graphs/make-falkner.pl mcl-14-137/graphs/make-falkner.pl
--- mcl-14-137.orig/graphs/make-falkner.pl	2007-07-17 11:20:33.000000000 +0200
+++ mcl-14-137/graphs/make-falkner.pl	2018-05-03 10:41:32.023704666 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 
 use Getopt::Long;
 use strict;
diff -ru mcl-14-137.orig/scripts/clmsnare mcl-14-137/scripts/clmsnare
--- mcl-14-137.orig/scripts/clmsnare	2007-07-02 12:16:01.000000000 +0200
+++ mcl-14-137/scripts/clmsnare	2018-05-03 10:42:09.148798258 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 
 #  (C) Copyright 2007 Stijn van Dongen
  #
diff -ru mcl-14-137.orig/scripts/clxdo mcl-14-137/scripts/clxdo
--- mcl-14-137.orig/scripts/clxdo	2014-04-25 12:04:38.000000000 +0200
+++ mcl-14-137/scripts/clxdo	2018-05-03 10:46:25.142547838 +0200
@@ -83,7 +83,7 @@
 require_num 1 "<cls-file>+"
 mx=$2
 mcxdump -imx $mx --no-values --dump-lines -o - \
-|  /usr/local/bin/perl -ane '$ct{@F-1}++; END { print map { "$_ $ct{$_}\n" } sort { $a <=> $b } keys %ct; }'
+|  /usr/bin/env perl -ane '$ct{@F-1}++; END { print map { "$_ $ct{$_}\n" } sort { $a <=> $b } keys %ct; }'
       ;;
 
 
@@ -116,7 +116,7 @@
 shift 2
 for mx in $@; do
       mcxdump -imx $mx --no-values --dump-lines -o - \
-   |  /usr/local/bin/perl -ane '$ct{@F-1}++; END {%i = map { ($_, 1) } grep { $_ >= $ENV{CLXDO_VAL1}; } keys %ct; ($lt, $n_lt, $gq, $n_gq) = (0,0,0,0); for my $x (keys %ct) { if ($i{$x}) { $n_gq += $ct{$x}; $gq += $x * $ct{$x}; } else { $n_lt += $ct{$x}; $lt += $x * $ct{$x} } } print "$n_lt $lt $n_gq $gq\n"; }'
+   |  /usr/bin/env perl -ane '$ct{@F-1}++; END {%i = map { ($_, 1) } grep { $_ >= $ENV{CLXDO_VAL1}; } keys %ct; ($lt, $n_lt, $gq, $n_gq) = (0,0,0,0); for my $x (keys %ct) { if ($i{$x}) { $n_gq += $ct{$x}; $gq += $x * $ct{$x}; } else { $n_lt += $ct{$x}; $lt += $x * $ct{$x} } } print "$n_lt $lt $n_gq $gq\n"; }'
 done
       ;;
 
diff -ru mcl-14-137.orig/scripts/docme mcl-14-137/scripts/docme
--- mcl-14-137.orig/scripts/docme	2013-05-16 12:32:13.000000000 +0200
+++ mcl-14-137/scripts/docme	2018-05-03 10:43:04.448448095 +0200
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/env perl -w
 
 #  This program matches
 #  A)
diff -ru mcl-14-137.orig/scripts/gengraph mcl-14-137/scripts/gengraph
--- mcl-14-137.orig/scripts/gengraph	2009-09-24 11:36:29.000000000 +0200
+++ mcl-14-137/scripts/gengraph	2018-05-03 10:42:52.678735459 +0200
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/env perl -w
 
 #  (C) Copyright 2009 Stijn van Dongen
  #
diff -ru mcl-14-137.orig/scripts/minimcl mcl-14-137/scripts/minimcl
--- mcl-14-137.orig/scripts/minimcl	2009-06-09 12:51:04.000000000 +0200
+++ mcl-14-137/scripts/minimcl	2018-05-03 10:42:42.452985126 +0200
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/env perl -w
 
 #  (C) Copyright 2006, 2007, 2008, 2009 Stijn van Dongen
  #
diff -ru mcl-14-137.orig/scripts/mlm-fifofum mcl-14-137/scripts/mlm-fifofum
--- mcl-14-137.orig/scripts/mlm-fifofum	2008-03-12 23:50:37.000000000 +0100
+++ mcl-14-137/scripts/mlm-fifofum	2018-05-03 10:41:50.300258445 +0200
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/env perl -w
 
 #  (C) Copyright 2007 Stijn van Dongen
  #
diff -ru mcl-14-137.orig/scripts/mlm-quilt mcl-14-137/scripts/mlm-quilt
--- mcl-14-137.orig/scripts/mlm-quilt	2009-05-20 00:30:25.000000000 +0200
+++ mcl-14-137/scripts/mlm-quilt	2018-05-03 10:42:58.911583281 +0200
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/env perl -w
 
 #  (C) Copyright 2007 Stijn van Dongen
  #
diff -ru mcl-14-137.orig/scripts/mlm-summary mcl-14-137/scripts/mlm-summary
--- mcl-14-137.orig/scripts/mlm-summary	2007-11-06 17:05:30.000000000 +0100
+++ mcl-14-137/scripts/mlm-summary	2018-05-03 10:41:43.416426514 +0200
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/env perl -w
 
 #  (C) Copyright 2007 Stijn van Dongen
  #
diff -ru mcl-14-137.orig/scripts/movgraph mcl-14-137/scripts/movgraph
--- mcl-14-137.orig/scripts/movgraph	2009-09-24 11:52:43.000000000 +0200
+++ mcl-14-137/scripts/movgraph	2018-05-03 10:42:00.405011738 +0200
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/env perl -w
 
 #  (C) Copyright 2009 Stijn van Dongen
  #
diff -ru mcl-14-137.orig/scripts/tree3.pl mcl-14-137/scripts/tree3.pl
--- mcl-14-137.orig/scripts/tree3.pl	2010-06-08 23:03:05.000000000 +0200
+++ mcl-14-137/scripts/tree3.pl	2018-05-03 10:42:34.168187403 +0200
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/env perl -w
 
 use strict;
 use Data::Dumper;
diff -ru mcl-14-137.orig/src/alien/oxygen/src/mcxdeblast mcl-14-137/src/alien/oxygen/src/mcxdeblast
--- mcl-14-137.orig/src/alien/oxygen/src/mcxdeblast	2009-03-24 17:09:46.000000000 +0100
+++ mcl-14-137/src/alien/oxygen/src/mcxdeblast	2018-05-03 10:41:22.716931889 +0200
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/env perl -w
 
 use Getopt::Long;
 
diff -ru mcl-14-137.orig/testing/stream/abc-pairs.pl mcl-14-137/testing/stream/abc-pairs.pl
--- mcl-14-137.orig/testing/stream/abc-pairs.pl	2008-10-28 23:37:10.000000000 +0100
+++ mcl-14-137/testing/stream/abc-pairs.pl	2018-05-03 10:40:42.597911375 +0200
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 use strict;
 use Getopt::Long;
